scripts/s3/s3api/object_operations.py

```diff
@@ -66,9 +67,12 @@ def __init__(self, access_key: str, secret_key: str, endpoint_url: str, test_id:
     # pylint: disable=too-many-locals
     async def execute_object_workload(self):
         """Execute object workload with given parameters."""
-        bucket = f'object-op-{self.test_id}-{time.perf_counter_ns()}'.lower()
-        self.log.info("Create bucket %s", bucket)
-        await self.create_bucket(bucket)
+        if not os.environ.get("options").get("degraded_mode"):
```

Comment:
You need to change this since you are not setting environment variable now.

Reply:
This comment is for all the os.environ instances in changed files.

Reply:
Done changes accordingly, Please check